packages/tree-view/src/__fixtures__/sampleTreeNodeObject.js

```diff
@@ -43,6 +45,7 @@ const sampleTreeNodeObject = [
           {
             id: 4,
             parentId: 2,
+            draggable: "true",
```

Comment:
does the draggable take both string and boolean?

Reply:
it can, since it's passed in as an HTML attribute, i think React translates things into strings as I think all HTML attributes take strings.